ghostjs-examples/test/fixtures/server.js

```diff
@@ -14,15 +14,24 @@ var Server = function(config  = {}) {
       throw err
     }
 
-    instance.route({
+    instance.route([{
+      method: 'GET',
+      path: '/longTime',
+      handler: function (req, reply) {
+        setTimeout(function() {
+          reply('Finally responding.')
+        }, 100000)
```

Comment:
I'm wondering if this route should take a timeout in the query string? Then maybe we can do other things with it.